com.mudita.mudita-center.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.mudita.mudita-center
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-desktop-file: muditacenter.desktop
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.gnome.SettingsDaemon
+  - --system-talk-name=org.freedesktop.login1
+modules:
+  - name: muditacenter
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/mudita/mudita-center/releases/download/2.5.0/Mudita-Center.AppImage
```

Comment:
so what happen when the aarch64 package is built?

why not building from source?

Reply:
There's precedent for repackaging appimage or deb files if that's what upstream provides, and building elecron apps from source in flatpak without network access is, in my limited experience, nearly impossible. This is closer to what the authors are delivering, simpler, and doesn't spend cpu time rebuilding an application that's already built.
If that's not acceptable I can look into building it from source.

Reply:
> There's precedent for repackaging appimage or deb files

Two wrongs don't make a right. There is also precedent for us SYSTEMATICALLY askinfg for source builds.

> and building elecron apps from source in flatpak without network access is, in my limited experience, nearly impossible.
 
Many do it. There is a lot of precedents too.

> and doesn't spend cpu time rebuilding an application that's already built.

This is 100% irrelevant to the question. Because if you want to talk about wasting CPU time, it's kinda the Electron pot calling the kettle black.